nuxeo-features/nuxeo-platform-ui-select2/src/main/java/org/nuxeo/ecm/platform/ui/select2/automation/SuggestUserEntries.java

```diff
@@ -191,6 +199,11 @@ public Blob run() {
                             }
                         }
                     }
+                    if (hidePowerUsersGroup) {
+                        if (POWERUSERS.equals(group.getId())) {
+                            break groupLoop;
```

Comment:
This keeps the style with the previous loop, but the previous look should have been written with `admins.contains()` in the first place so we wouldn't have this discussion.